plugins/modules/cloud/lxc/lxc_container.py

```diff
@@ -1380,7 +1365,7 @@ def _create_tar(self, source_dir):
             '.'
         ]
 
-        rc, stdout, err = self._run_command(
+        rc, stdout, err = self.module.run_command(
             build_command=build_command,
             unsafe_shell=True
```

Comment:
This command is supposed to have `use_unsafe_shell=True`, which is only used when `args` is a string and not a list. Also there is no option `build_command` for `module.run_command()`:
```suggestion
            ' '.join(build_command),
            use_unsafe_shell=True
```

Reply:
In that case this needs to be changed to:
```suggestion
            build_command
```

Reply:
Yep, that's where I was heading to.